src/main/java/address/sync/model/CloudPerson.java

```diff
@@ -0,0 +1,117 @@
+package address.sync.model;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+public class CloudPerson {
+    private int id;
+    private String firstName;
+    private String lastName;
+    private String street;
+    private String city;
+    private String postalCode;
+    private List<CloudTag> tags;
+    private boolean isDeleted;
+    private LocalDateTime lastUpdatedAt;
+
+    public CloudPerson() {
+        setLastUpdatedAt(LocalDateTime.now());
```

Comment:
This should be removed since the empty constructor should not be used aside from reading and writing into XML files. Same for `CloudTag`.